EIPS/eip-slot-precompile.md

```diff
@@ -0,0 +1,60 @@
+---
+eip:
```

Comment:
```suggestion
eip: 7843
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.